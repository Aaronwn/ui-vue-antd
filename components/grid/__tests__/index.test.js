import { mount } from '@vue/test-utils'
import { Col, Row } from '..'

describe('Grid', () => {
  it('should render Col', () => {
    const wrapper = mount(Col, {
      propsData: {
        span: 2,
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should render Row', () => {
    const wrapper = mount(Row)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
