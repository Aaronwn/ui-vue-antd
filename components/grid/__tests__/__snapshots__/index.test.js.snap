// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render Col 1`] = `<div class="ant-col-2"></div>`;

exports[`Grid should render Row 1`] = `<div class="ant-row"></div>`;
